docs/newpage.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
```

Comment:
I thought the breadcrumbs and side nav on the `newpage.html` was confusing.

I guess it's only an example, but `newpage.html` is a parent page. So the breadcrumbs should just be:

Home > New Page.

Same as the side-nav.

The way you did sidenav and breadcrumbs on the landing page and subpages made a lot of sense though :)